src/app.js

```diff
@@ -6,7 +6,10 @@ p5.prototype.getAudioContext = getAudioContext;
 p5.prototype.userStartAudio = userStartAudio;
 
 import './master';
-import './helpers';
+
+import { freqToMidi } from './helpers';
+p5.prototype.freqToMidi = freqToMidi;
```

Comment:
I'm confused why this change is happening in this PR? There are a bunch of methods that get attached to the p5.prototype as part of helpers, let's be sure to add them all (when `helpers` is addressed)